calcom/cal.com
# Fixes some translation issues

## What does this PR do?

Fixes some translation issues in rescheduling email in french (#5385) and german. Fixes the variable name {{teamName}} to {{team}} in team invitation email in several languages. 

Before: 
![Screenshot 2022-12-02 at 18 40 40](https://user-images.githubusercontent.com/30310907/205352818-90218265-c89e-4c9e-8ef8-8673b5923a86.png)

![Screenshot 2022-12-02 at 18 41 26](https://user-images.githubusercontent.com/30310907/205352914-4a964da3-2210-405a-942b-ec14a0a87f80.png)


After
![Screenshot 2022-12-02 at 18 41 09](https://user-images.githubusercontent.com/30310907/205352869-849dc6e3-2bfa-439c-8304-1ca22173d15f.png)

![Screenshot 2022-12-02 at 18 41 47](https://user-images.githubusercontent.com/30310907/205352975-2c2d9fba-2c54-41f0-aaf9-df50f0393231.png)


## How should this be tested?

- Check rescheduling email in french 
- Check team invitation email in french
